benches/is_enabled.rs

```diff
@@ -179,6 +177,7 @@ fn client(count: usize) -> client::Client<http_client::native::NativeClient, Use
                 name: "flexibleRollout".into(),
                 parameters: Some(hashmap!["stickiness".into()=>"DEFAULT".into(),
```

Comment:
DEFAULT => "default" ?

Reply:
ah, missed one :). thanks. I wonder if we can do something to make this more typesafe.